### ISSUE TITLE
App store deployment error

### ISSUE THREAD

**Issue by ClemPrt** [FlutterFlow/flutterflow-issues#3263]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When I deploy my app on the App Store, I get the following error message.

"Failed Step: Flutter build ipa and automatic versioning

Found build number 67 from TestFlight version 2.0.7

Archiving com.mycompany.livetastar...
Upgrading contents.xcworkspacedata
Updating project for Xcode compatibility.
Upgrading project.pbxproj
Upgrading Runner.xcscheme
Upgrading Info.plist
Removing script build phase dependency analysis.
Adding input path to Thin Binary build phase.
Automatically signing iOS for device deployment using specified development team in Xcode project: W2A8RF2C94
Running pod install...                                              6.9s
Running Xcode build...                                          
Xcode archive done.                                         242.4s
Failed to build iOS app
Error (Xcode): Target release_ios_bundle_flutter_assets failed: IconTreeShakerException: ConstFinder failure: Unhandled exception:

Encountered error while archiving for device.

Build failed :|
Step 10 script `Flutter build ipa and automatic versioning` exited with status code 1"

### Expected Behavior

That the deployment works ;)

### Steps to Reproduce

1. Deploy with version code on App Store

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4ojMrq05Ngrt9Y+KX2KfpahSQsQzN5aOcn0spOGDwbfaKvPIlzaeDeUEhDXd+8TEFXKEWKpHMx+dbokvHpNPQFEzuVb4ROyblcVQ/ifGW/aoS0C6uOWUFCTMZgfGrF38G7mBQlVe9sR04gw1Otf+6AcCjqNI/7Zwh9f6PHaOI=

### Visual documentation

<img width="789" alt="Capture d’écran 2024-06-27 à 15 33 12" src="https://github.com/FlutterFlow/flutterflow-issues/assets/24780295/e6cb1919-12be-4608-9e8e-10b6fd3a168a">

### Environment

```markdown
- FlutterFlow version: 4.1.63+
- Platform: MAC OS DESKTOP
- Browser name and version: Chrome 126.0.6478.116
- Operating system and version affected: SONOMA 14.5 (23F79)
```

**Comment by ignalauret** [comment]
Hey @ClemPrt thanks for your report. I can see a possible fix for this, but I will send it to our technical team so they can take a second look. We will keep you updated here, have a great day!

**Comment by Alezanello** [comment]
Hello!

First of all, I apologize for the late response.

I wanted to check if the recent FlutterFlow updates have resolved your issue or if it still persists. Additionally, are you deploying through FlutterFlow, or are you using an external service like GitHub or Codemagic for your app's compilation?

Thank you for your patience and understanding.

Best regards,<br>Azanello

**Comment by ClemPrt** [comment]
Hello, I've tried several times today and it's still the same. And I do everything via FlutterFlow 

<body style="background-color:#14161C">
<style>
body {
  color:#C8C8C8;
  overflow-x: hidden;
  padding-top: 0px;
  padding-right: 5px;
  padding-bottom: 5px;
  padding-left: 5px;
  font-family: monaco;
}
h4 {
  margin-bottom: 4px;
  font-weight: bold;
  color: #DF3F3F;
}
</style>
<h4>Failed Step: Flutter build ipa and automatic versioning</h4>
<div style="white-space: pre-wrap;line-height: 1.5">
<span style="color:#859900">Found build number 67 from TestFlight version 2.0.7</span>

Archiving com.mycompany.livetastar...
Upgrading contents.xcworkspacedata
Updating project for Xcode compatibility.
Upgrading project.pbxproj
Upgrading Runner.xcscheme
Upgrading Info.plist
Removing script build phase dependency analysis.
Adding input path to Thin Binary build phase.
Automatically signing iOS for device deployment using specified development team in Xcode project: W2A8RF2C94
Running pod install...                                              4.2s
Running Xcode build...                                          
Xcode archive done.                                         166.1s
Failed to build iOS app
Error (Xcode): Target release_ios_bundle_flutter_assets failed: IconTreeShakerException: ConstFinder failure: Unhandled exception:

Encountered error while archiving for device.


Build failed :|
Step 10 script `Flutter build ipa and automatic versioning` exited with status code 1
</div>
</body>

**Comment by msusviela** [comment]
Hi @ClemPrt. I was not able to reproduce this issue from your project due to credentials restrictions. <br>Please let me know if you tried these possible solutions:

1. Use this flag while compiling:<br>--no-tree-shake-icons
2. Check if you upgraded manually the version of the build.

**Comment by ClemPrt** [comment]
Yes, I've tried both automatic and manual builds. And I can't use the "--no-tree-shake-icons" because I upload directly with FlutterFlow

**Comment by ClemPrt** [comment]
With the new version 4.1.70+, it still doesn't work! Still the same problem with the same error message. Do you have any suggestions?

**Comment by ClemPrt** [comment]
The problem still hasn't been fixed with the 4.1.73 update! Do you have any information on how to fix the problem?

**Comment by msusviela** [comment]
I'm sorry but we are unable to reproduce this issue from a blank project (even from the new version 4.1.74), can you please reach out to [support@flutterflow.io](mailto:support@flutterflow.io)? They can better assist you with ensuring your project is configured correctly
